src/Components/ContextMenu/index.css

```diff
@@ -16,3 +17,7 @@
   padding: 0px;
   cursor: pointer;
 }
+
+.context-menu > ul > li:hover {
+  color: green;
+}
```

Comment:
Newline is missing, however I thought about applying some basics with EditorConfig when I will work on my stuff, so no biggie.